### PR TITLE
feat(tests): added allure report for tests

### PR DESCRIPTION
**Pull request by nestoragent**
# Contribution checklist

## Brief description
feat(tests): added allure report for tests
Снимок экрана 2023-11-09 в 22 58 23

Снимок экрана 2023-11-06 в 23 24 59
Снимок экрана 2023-11-07 в 23 52 15
Снимок экрана 2023-11-07 в 23 52 07

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -202,6 +198,28 @@ jobs:
         with:
           name: playwright-results
           path: ./tests/sanity/playwright-report/
+      - name: Generates Allure Report
+        uses: simple-elf/allure-report-action@master
+        if: always()
```
Comment (haiodo) [inline .github/workflows/main.yml:203]: I think we need to enable it only for main branch

Comment (nestoragent) [reply .github/workflows/main.yml:203]: yes, agree

Comment (nestoragent) [reply .github/workflows/main.yml:203]: done

Comment (haiodo) [reply .github/workflows/main.yml:203]: Why we still use "if: always()"

Comment (nestoragent) [reply .github/workflows/main.yml:203]: it's for generating the report, which we can also download to the local.
Deployment executing by the - name: Deploy report to Github Pages step
it used the 
if: ${{ github.ref == 'refs/heads/main' }}
param

**Review by haiodo — APPROVED** [review]

**Pull request merged**